ui/src/kapacitor/components/RuleMessageAlertConfig.js

```diff
@@ -14,11 +14,12 @@ const RuleMessageAlertConfig = ({updateAlertNodes, alert, rule}) => {
     return null
   }
   return (
-    <div className="rule-section--item alert-message--config">
+    <div className="rule-section--row rule-section--border-bottom">
       <p>{DEFAULT_ALERT_LABELS[alert]}</p>
       <input
         id="alert-input"
-        className="form-control size-486 form-control--green input-sm"
+        className="form-control input-sm form-malachite"
+        style={{flex: '1 0 0'}}
```

Comment:
Starting to think I should make a utility class for this kinda stuff since I find myself doing this over and over again